app/src/main/java/net/gsantner/opoc/ui/FilesystemViewerAdapter.java

```diff
@@ -331,6 +331,10 @@ public boolean areItemsSelected() {
         return _currentSelection;
     }
 
+    public File descriptionFile() {
```

Comment:
This is not needed. For selection there is getCurrentSelection(), use that please and delete this one

Reply:
Fix in wladokacur/markor@6c6f39893eb4315466faf99062318f8da95c45a8